tests/test_slack.py

```diff
@@ -858,6 +858,72 @@ def test_private_user(self):
 
         self.assertEqual(user, '{"ok":false,"user":null}')
 
+    def test_metadata_id(self):
+
+        client = Slack('CHANNEL', 'TOKEN')
+
+        # When 'user', 'bot_id', 'comment', or 'username' do not exist.
+        item = {
+            'type': 'message',
+            'text': 'A file was commented on',
+            'ts': '1529335169.000726',
+            'channel_info': {
+                'id': 'CHANNEL',
+                'name': 'test',
+                'is_channel': True,
+                'num_members': 4
+            }
+        }
+        metadata_id = client.metadata_id(item)
+        expected_id = item['ts']
+
+        self.assertEqual(metadata_id, expected_id)
+
+        # When 'user', 'bot_id', 'comment', or 'username' do not exist.
+        item = {
+            'type': 'message',
+            'text': 'A file was commented on',
+            'ts': '1529335169.000726',
+            'channel_info': {
+                'id': 'CHANNEL',
+                'name': 'test',
+                'is_channel': True,
+                'num_members': 4
+            }
+        }
+        metadata_id = client.metadata_id(item)
+        expected_id = item['ts']
+
+        self.assertEqual(metadata_id, expected_id)
```

Comment:
Just a minor comment, I think this test is the same as the previous one.

Reply:
ups